sql/flaky-test-ratio.sql

```diff
@@ -0,0 +1,14 @@
+SELECT format_datetime(runs.created_at, 'Y:w') year_week
+     , jobs.run_id
+     , runs.created_at
+     , jobs.name
+     , count(*) FILTER (WHERE (jobs.conclusion = 'success')) success_count
+, count(*) FILTER (WHERE (jobs.conclusion = 'failure')) failure_count
+, round(((CAST(count(*) FILTER (WHERE (jobs.conclusion = 'failure')) AS double) / CAST((count(*) FILTER (WHERE (jobs.conclusion = 'success')) + count(*) FILTER (WHERE (jobs.conclusion = 'failure'))) AS double)) * 100), 2) flaky_ratio_percent
+FROM jobs
+         JOIN runs ON jobs.run_id = runs.id
+WHERE NOT jobs.name LIKE 'Test Report%'
+GROUP BY 1, jobs.run_id, runs.created_at, jobs.name
+HAVING ((count(*) FILTER (WHERE (jobs.conclusion = 'success')) > 0) AND (count(*) FILTER (WHERE (jobs.conclusion = 'failure')) > 0))
+ORDER BY year_week, 6 DESC, jobs.name, jobs.run_id
```

Comment:
Let's use only colum index numbers here. If you'd rearrange the columns in SELECT to match the order here, it shouldn't reduce the readability of the query. The column order here is significant, that is for example the `flaky_ratio_percent` (2nd) is more important to the reader than the job name and run id (last ones).